clio-integration-test/src/it/resources/logback-test.xml

```diff
@@ -0,0 +1,24 @@
+<configuration>
```

Comment:
I couldn't find a quick-and-easy solution for this, so I'm going to file a separate ticket for general configuration cleanup.